libs/ui-toolkit/src/components/icon/vega-icons/svg-icons/icon-streak.tsx

```diff
@@ -0,0 +1,26 @@
+export const IconStreak = ({ size = 14 }: { size: number }) => {
+  return (
+    <svg width="14" height="14" viewBox="0 0 14 14" fill="fillCurrent">
```

Comment:
All the fill attributes are not required. It will be applied by the fill-current class in vega-icon.tsx

Reply:
removed